src/State/FeedProvider.php

```diff
@@ -49,17 +43,27 @@ public function toOutput(object $object): object
     {
         assert($object instanceof Feed);
 
+        $id = $object->getId();
+        $modifiedAt = $object->getModifiedAt();
+        $slide = $object->getSlide();
+        $feedSource = $object->getFeedSource();
+
+        assert(null !== $id);
+        assert(null !== $modifiedAt);
+        assert(null !== $slide);
+        assert(null !== $feedSource);
+
         $output = new \App\Dto\Feed();
```

Comment:
DTOs are usually imported with aliases (e.g. https://github.com/os2display/display-api-service/blob/develop/src/State/SlideProvider.php#L9).

Consider adding and using

```php
use App\Dto\Feed as FeedDTO;
```

for consistency.